id/news/_posts/2019-11-23-ruby-2-7-0-preview3-released.md

```diff
@@ -0,0 +1,371 @@
+---
+layout: news_post
+title: "Ruby 2.7.0-preview3 Dirilis"
+author: "naruse"
+translator: "meisyal"
+date: 2019-11-23 12:00:00 +0000
+lang: id
+---
+
+Kami dengan senang hati mengumumkan rilis dari Ruby 2.7.0-preview3.
+
+Sebuah versi *preview* dirilis untuk mengumpulkan umpan balik rilis final
+yang direncanakan rilis pada bulan Desember. Versi ini dirilis terutama untuk
+mengkonfirmasi *compatibility* dari *keyword argument*.
```

Comment:
instead of compatibility, how about "kompatibilitas"?

Reply:
I thought "kompatibilitas" is not a valid word in Bahasa. It's available on KBBI https://kbbi.web.id/kompatibilitas.

OK. I will translate all the "compatibility" to "kompatibilitas".

Reply:
I thought "kompatibilitas" was not a valid word in Bahasa. It's available on KBBI https://kbbi.web.id/kompatibilitas.

OK. I will translate all the "compatibility" to "kompatibilitas".

Reply:
Translated to "kompatibilitas" @kuntoaji